### ISSUE TITLE
How to generate app clip code image?

### ISSUE THREAD

**Issue by pot2mayo** [aheze/AppClipQuickStart#2]
This is more of a question than an issue.

When I access the site `https://getfind.app` I can indeed see the app clip banner popping up.
However I was wondering how to access it using a app clip image.

I tried to generate one using Apple official tool and the following command:

`AppClipCodeGenerator generate --url https://getfind.app --index 9 --output ~/Documents/getfind-clip.svg`

Which gives me this image:

![image](https://user-images.githubusercontent.com/5609225/149288415-5043aeb8-8504-44d1-8511-bc39b685bcfc.png)

However pointing at it with the iPhone camera just gives me a `no usable data found` message and nothing happens :(

Can you give some info on how to generate the actual QR code image please?

Great sample code by the way, really helped!

**Comment by aheze** [comment]
Sorry for the late response. This might help: https://github.com/alfianlosari/AppClipCodeGenerator, but @pot2mayo have you found a solution of your own?

**Comment by pot2mayo** [comment]
@aheze 
Thanks for your reply
No I could not find any QR code that triggers the app clip.
In the case of https://getfind.app that you mention, which seems to be a perfectly working app clip when starting it from the website, even using the Apple official app clip generator I could not get a QR that works.

Actually the one I posted above in my original message is using this official tools and it encodes the https://getfind.app URL.

I also checked the Apple App App Site Association (AASA) of the website ([here](https://getfind.app/.well-known/apple-app-site-association)) and the clip is properly defined.

Really no idea how app clip gets triggered 🤷🏼‍♂️

**Comment by aheze** [comment]
Yeah me neither. I kind of gave up with App Clips. When I need them again I'll probably file a developer support ticket with Apple and post the results here.

**Comment by pot2mayo** [comment]
I see 😅 This is crazy that it's that hard to figure out how those QR codes work.

Here is an example
![image](https://user-images.githubusercontent.com/5609225/158496621-34ea0a4c-57fd-4e5c-9973-51ef4b27899e.png)

Pointing your iPhone at it will instantly show up a link to the app clip.

The source website https://chibi.app/ also has a banner and [here](https://chibi.app/.well-known/apple-app-site-association) is the AASA
